lib/diagnostic.js

```diff
@@ -17,7 +17,19 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+Description: Calender program for keeping track of things
+Entity: Event
+Attributes: when, where, duration
+Methods: conflictingEvent
+
+Entity: User
+Attributes: name, address, e-mail
+Methods: schedule, outOfTown
```

Comment:
`outOfTown` sounds like a property (and, as above, it should probably be called `isOutOfTown` since it'd be giving us a boolean). Why do you think it makes sense for it to be a method? Is it performing some kind of calculation to determine if the user is out of town?